package com.rakovets.course.java.core.practice.exception_handling;

public class Task02 {

    public  static void throwArrayExeption(){
        int[] ar = new int[3];
        ar[3] = 3;
    }
}
